aosp_diff/celadon_ivi/packages/services/Car/0006-Force-stop-bootanimation-when-CarLauncher-UI-ready.patch

```diff
@@ -0,0 +1,69 @@
+From bc84d40529e434ea6645dc74e6f2c26f75dcc4f4 Mon Sep 17 00:00:00 2001
+From: Zhou Jingyu <jingyu.zhou@intel.com>
+Date: Thu, 11 May 2023 12:40:25 +0800
+Subject: [PATCH] Force stop bootanimation when CarLauncher UI ready
```

Comment:
since it is a workaround, we should add [WA] in patch subject.

Reply:
Just fixed, please review again